src/server/components/dashboard/annualReview/controllers.ts

```diff
@@ -63,6 +66,11 @@ async function confirmNewAnnualReviewDate(req: Request, res: Response): Promise<
     return res.redirect(`${res.locals.listsEditUrl}/annual-review-date`);
   }
 
+  const roles = req.user?.userData?.jsonData.roles;
+  if (roles && !roles.includes(UserRoles.Administrator)) {
```

Comment:
req.user is this class `src/server/components/auth/authenticated-user.ts`. I've already added a getter `isAdministrator`. so you can do `if(!req.user.isAdministrator)`

I think this would also probably be better as middleware. 


```

function allowAdminsOnly(req, res, next) {
  if(req.user.isAdministrator) {
    return next() // "editDateController" or "editDatePostController"
  }

  req.flash("error", "You do not have permissions to edit the annual review date");
  return res.redirect(res.locals.listsEditUrl);

}

listRouter.get("/:listId/annual-review-date", allowAdminsOnly, annualReview.editDateGetController);
listRouter.post("/:listId/annual-review-date", allowAdminsOnly, annualReview.editDatePostController);

```